src/validations/profileSchema.js

```diff
@@ -0,0 +1,95 @@
+import { check } from 'express-validator/check';
+import makeUpperCase from '../helpers/makeUpperCase';
+
+const profileSchemas = {
+  profileUpdateSchema: [
+    check('userName')
+      .optional()
+      .trim()
+      .not()
+      .isEmpty({ ignore_whitespace: true })
+      .withMessage('userName cannot be blank')
+      .isLength({ min: 2, max: 20 })
+      .withMessage('username must be at least 2 characters, and maximum 20')
+      .not()
+      .matches(/^[A-Za-z]+[-]{1}[A-Za-z]+([-]{1}[A-Za-z]+)+$/, 'g')
```

Comment:
I'm a bit confused as to what's going on here. Also, isn't it better to use the same validation that was done for username when creating it?